src/alfrescoApi.ts

```diff
@@ -72,13 +72,13 @@ export class AlfrescoApi implements Emitter, AlfrescoApiType {
 
         this.clientsFactory();
 
-        this.processClient = new ProcessClient(this.config, this.httpClient);
```

Comment:
Why was it here originally?

Reply:
the `ProcessClient` instance is already created by the  this.clientsFactory();` method above
https://github.com/Alfresco/alfresco-js-api/pull/1545/files/d3f0b087a109dc00df0fa2e4c6a2bbebede015e6#diff-f9c402c30455ebed6bc905c9840dc623003cc816818179c2676b6afbd298054eL166

Reply:
I removed it because it's duplicated